app/views/home/policy.html.erb

```diff
@@ -0,0 +1,134 @@
+
+        <h2 class="disclosures__title">We use cookies to:</h2>
+        <!-- fix indentation of h2 -->
+        <li>Understand and save user&apos;s preferences for future visits.</li>
+          You can choose to have your computer warn you each time a cookie is being sent, or you can choose to turn off all cookies. You do this through your browser settings. Since browser is a little different, look at your browser&apos;s Help Menu to learn the correct way to modify your cookies.
+        <h2 class="disclosures__title">  If users disable cookies in their browser: </h2>
+        <!-- fix indentation of h2 as well as margin right -->
+          <p>If you turn cookies off, Some of the features that make your site experience more efficient may not function properly. Some of the features that make your site experience more efficient and may not function properly.
+          </p>
+       </div>
+
+
+       <div class="col-md-12 disclosures__information">
+         <h1 class="disclosures__one-line">How does our site handle Do Not Track signals?</h1>
+          <p>We honor Do Not Track signals and Do Not Track, plant cookies, or use advertising when a Do Not Track (DNT) browser mechanism is in place.</p>
+       </div>
+
+       <div class="col-md-12 disclosures__information">
+         <h1 class="disclosures__one-line">Does our site allow third-party behavioral tracking?</h1>
+          <p>It's also important to note that we do not allow third-party behavioral tracking.</p>
+       </div>
+
+
+       <div class="row col-md-12 disclosures-bottom">
+         <div class="col-md-12">
+           <hr class="disclosures-bottom__line">
+         </div>
+       </div>
+
+       <div class="col-md-12 disclosures__information">
+         <h1 class="disclosures__one-line">Third-party disclosure</h1>
+           <p>We do not sell, trade, or otherwise transfer to outside parties your Personally Identifiable Information unless we provide users with advance notice. This does not include website hosting partners and other parties who assist us in operating our website, conducting our business, or serving our users, so long as those parties agree to keep this information confidential. We may also release information when it's release is appropriate to comply with the law, enforce our site policies, or protect ours or others' rights, property or safety.
```

Comment:
This `<p>` tag is missing a closing tag.

Reply:
good catch!